docs/ibc/proto-docs.md

```diff
@@ -1555,6 +1557,34 @@ MsgRegisterAccountResponse defines the response for Msg/RegisterAccount
 
 
 
+
+<a name="ibc.applications.interchain_accounts.controller.v1.MsgSubmitTx"></a>
+
+### MsgSubmitTx
+MsgSubmitTx defines the payload for MsgSubmitTx
+
+
+| Field | Type | Label | Description |
+| ----- | ---- | ----- | ----------- |
+| `owner` | [string](#string) |  |  |
+| `connection_id` | [string](#string) |  |  |
+| `timeout` | [string](#string) |  |  |
```

Comment:
should timeout be Height here?

Reply:
I think we should support both. Should be similar to `MsgTransfer`

Reply:
I agree, I think we should support both going forward but the current implementation uses [`clienttypes.ZeroHeight`](https://github.com/cosmos/ibc-go/blob/main/modules/apps/27-interchain-accounts/controller/keeper/relay.go#L66).
I think we can only support the timeout height via the new execution path - i.e. existing auth modules using the `SendTx` go API directly won't be able to specify a height, right? 

For `SubmitTx` we can use `uint64` for `timeout_timestamp` and `ibc.core.client.v1.Height` for `timeout_height` like [`MsgTransfer`](https://github.com/cosmos/ibc-go/blob/main/proto/ibc/applications/transfer/v1/tx.proto#L36-L40) as @colin-axner  suggests! :)